bitbucket_issue_tracker.py

```diff
@@ -0,0 +1,61 @@
+from rake_nltk import Rake, Metric
+import argparse
+import requests 
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--only_title', default=False, action='store_true', help="For extracting keywords only from title of the Bitbucket issue.")
+parser.add_argument('--with_comments', default=False, action='store_true', help="For extracting keywords from the title, content (body) and comments of the Bitbucket issue.")
+parser.add_argument('--min_score', default=4, help="Minimum required score of an extracted keyword.")
+parser.add_argument('--min_keyw_length', default=1, help="Minimum length of an extracted keyword phrase.")
+parser.add_argument('--max_keyw_length', default=4, help="Maximum length of an extracted keyword phrase.")
+parser.add_argument('--sort_by_updated', default=False, action='store_true', help="To sort issues based on updation date")
+parser.add_argument('--num_pages', default=1, help="Number of pages for which results will be showed. (Each page shows 25 results)")
+args = parser.parse_args()
+
+for page in range(int(args.num_pages)):
+    rake = Rake(min_length = args.min_keyw_length, max_length= args.max_keyw_length, ranking_metric=Metric.WORD_DEGREE)
+    if not args.sort_by_updated:
+        r = requests.get(url = "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues?sort=-votes&page=" + str(page+1)) 
+    else:
+        r = requests.get(url = "https://api.bitbucket.org/2.0/repositories/osrf/gazebo/issues&page=" + str(page+1)) 
+        
+    data = r.json()
+    for i,j in enumerate(data['values']):
```

Comment:
I found a problem while calling the script with the `sort_by_updated` flag. Let me know if you can not reproduce it:
```
(bionic)jrivero@nium gz-bigindex $ (issue_tracker) python3 bitbucket_issue_tracker.py --sort_by_updated
Traceback (most recent call last):
  File "bitbucket_issue_tracker.py", line 23, in <module>
    for i,j in enumerate(data['values']):
KeyError: 'values'
```

Reply:
Invalid URL, my bad. 
Fixing this in the next commit. Also raising error for unsuccessful request.